src/cli/scaffold/rust/lib.rs

```diff
@@ -0,0 +1,4 @@
+#[macro_use]
+extern crate hdk;
+
+// TODO: make this a nice friendly  boilerplate
```

Comment:
I'm going to add the issue number here, which is a pattern we're trying to keep up. Then, we can leave that issue open, to actually fulfill this boilerplate